### PR TITLE
[No ticket] Update FontAwesome styling issues

### PR DESCRIPTION
**Pull request by futa-ikeda**
- Ticket: [NA]
- Feature flag: n/a

## Purpose
- Update styling of the draft registration left-nav
  - Grey line was off-center
  - Grey line was visible behind the checkmark and exclamation icons
- Update styling of Registries navbar
  - Change color of down-arrow when hovering and expanding dropdown to navigate to other OSF Applications (left side of navbar)
  - Change color of down-arrow when hovering and expanding user gravatar menu (right side of navbar)
- Update styling of comment card
  - Change style of asterisk showing that a comment was modified

## Summary of Changes
- Update the css selectors in these files to match what is expected in the newer version of FontAwesome
- Remove unneeded style in draft registration left-nav

### Draft registrations leftnav
Old:
Screen Shot 2021-05-28 at 3 44 08 PM

New:
Screen Shot 2021-05-28 at 3 43 32 PM

### Registries navbar
Old:
Screen Shot 2021-05-28 at 4 07 30 PM

New:
Screen Shot 2021-05-28 at 4 07 59 PM

### Comment card
Old:
Screen Shot 2021-05-28 at 4 09 55 PM

New:
Screen Shot 2021-05-28 at 4 11 18 PM

## Side Effects
- None

## QA Notes
- These changes are purely stylistic changes and shouldn't effect other functionality. These are also features that are in Registries as well.
- I don't know if there is a way to add comments on a registration currently, so the updates to the comment-card may not have been needed? But in case we use that component again in the future, it's been updated 🤷

### CODE REVIEW

File: lib/registries/addon/overview/-components/overview-topbar/template.hbs [hunk 1]
```diff
@@ -1,9 +1,11 @@
-<div data-test-topbar-states>
-    <RegistriesStates
-        @registration={{@registration}}
-        @isModeratorMode={{@isModeratorMode}}
-    />
-</div>
+{{#unless @registration.isAnonymous}}
```
Comment (fabmiz) [inline lib/registries/addon/overview/-components/overview-topbar/template.hbs:1]: Hmm, isn't this already in develop?

File: tests/engines/registries/acceptance/overview/topbar-test.ts [hunk 2]
```diff
@@ -74,6 +74,17 @@ module('Registries | Acceptance | overview.topbar', hooks => {
         assert.dom('[data-test-topbar-share-bookmark-fork]').isVisible();
         assert.dom('[data-test-topbar-states]').isVisible();
 
+        const anonymousReg = server.create('registration', {
```
Comment (fabmiz) [inline tests/engines/registries/acceptance/overview/topbar-test.ts:77]: ditto ☝🏽

**Review by fabmiz — COMMENTED** [review]
drive-by: looks great 💯

**Review by adlius — COMMENTED** [review]
LGTM. Some merge conflicts. Also need to check why the changes compared doesn't pick up what's already in develop.

**Review by fabmiz — APPROVED** [review]
💯

**Pull request merged**